src/context.cpp

```diff
@@ -343,8 +343,8 @@ namespace librealsense
             std::copy(begin(tm2_devices), end(tm2_devices), std::back_inserter(list));
         }
 #endif
-
-        if (mask & RS2_PRODUCT_LINE_D400 || mask & RS2_PRODUCT_LINE_SR300)//supported recovery devices
+        // Supported recovery devices
+        if (mask & RS2_PRODUCT_LINE_D400 || mask & RS2_PRODUCT_LINE_SR300 || mask & RS2_PRODUCT_LINE_L500) 
```

Comment:
How did it work before?

Reply:
The viewer always query with mask of ANY(0xFF)
If a user will query with L515 mask he will not see a recovery device before the change.
Now it fixed